### DISCUSSION TITLE
4.5.0 version for arm64

### DISCUSSION THREAD

**Discussion by DiagnozHub** [Motion-Project/motion#1603] [General]
Hey )

I cant find focal_motion_4.5.0-1_arm64.deb in last release.

Can you do the bild ? )

I tried self (using instruction) but I got errors like: /usr/bin/ld: /lib/aarch64-linux-gnu/libsndio.so.7.0: undefined reference to `snd_pcm_sw_params_set_period_event@ALSA_0.9'

I tried inside do it inside arm device https://www.khadas.com/vim3

**Comment by Mr-Dave** [comment]
There is limited ability to do arm builds due to limited hardware and even variations among the emulators.  

I'd note that the undefined referenced you've indicated is "unusual".  snd and ALSA would typically only be a ffmpeg component rather than Motion.  Something much larger than Motion seems to be an issue.

**Reply by DiagnozHub** [reply]
Release [Release 4.4.0](https://github.com/Motion-Project/motion/releases/tag/release-4.4.0) has build for ARM64: focal_motion_4.4.0-1_arm64.deb

Release [Release 4.5.0](https://github.com/Motion-Project/motion/releases/tag/release-4.5.0) has no build for ARM64.
I think tere is no problems just buld if you have envirenment for it )

**Comment by tosiara** [comment]
I'd suggest to make 4.5.1 release and then I could spawn my build machine and build any kind of deb and update the release with the artifacts

**Reply by DiagnozHub** [reply]
Brilliant suggestion ))
When we could wait it ?

**Reply by tosiara** [reply]
@Mr-Dave will handle the new release as soon as he feels it is time for release. Just be patient

**Reply by Mr-Dave** [reply]
The minor release will likely be in a few days.  I'm waiting to see if any other issues come in and then probably this weekend.

**Reply by tosiara** [reply]
Sounds good. Just do the release and let me build all the debs

**Comment by tosiara** [comment]
The release has been created, I'm about to run the build machine. Will publish DEBs soon

**Comment by tosiara** [comment]
Almost all builds have been uploaded, only missing Bionic/Focal ARM64. Created an issue: https://github.com/Motion-Project/motion-packaging/issues/18
